addons/website/static/src/components/dialog/seo.js

```diff
@@ -433,7 +433,7 @@ export class OptimizeSEODialog extends Component {
             }
         }
         data.website_meta_og_img = seoContext.metaImage;
-        await this.orm.write(this.object.model, [this.object.id], data);
+        await this.orm.write(this.object.model, [this.object.id], data, {lang: this.website.currentWebsite.metadata.lang});
```

Comment:
Not working. I guess it was tested but is not working anymore
need to be
```js
        await this.orm.write(this.object.model, [this.object.id], data, {
            context: {lang: this.website.currentWebsite.metadata.lang},
        });
```
![image](https://user-images.githubusercontent.com/30048408/194777782-f9453eb0-b591-498e-8810-37453a55f57c.png)